dask/dataframe/core.py

```diff
@@ -2438,9 +2438,11 @@ def var(
         axis = self._validate_axis(axis)
         _raise_if_object_series(self, "var")
         with check_numeric_only_deprecation(), check_nuisance_columns_warning():
-            meta = self._meta_nonempty.var(
-                axis=axis, skipna=skipna, numeric_only=numeric_only
-            )
+            with warnings.catch_warnings():
+                warnings.filterwarnings("ignore", category=RuntimeWarning)
```

Comment:
What warning it this? I recommend adding a `message` that this needs to match against to avoid unintentionally catching a different `RuntimeWarning` than the one that's intended. 

Also, it looks like we're wanting to catch the same warning (I think) multiple times. A little utility like 

https://github.com/dask/dask/blob/b86786e3e1bd5be37a24d5b9743c6c043b9daffc/dask/dataframe/_compat.py#L109-L120

(see the two context managers being used in the line above) would be useful

Reply:
This is raising multiple warnings right now unfortunately. But I fixed this upstream in pandas over the weekend and will be released in 2.0.1, so if we are ok with this warning on 2.0 only, we can remove the ``catch_warnings`` statement.

Reply:
Added a utility to catch warnings for 2.0.0